src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Object/ObjectDefaultConverter.cs

```diff
@@ -4,13 +4,14 @@
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Runtime.CompilerServices;
+using System.Text.Json.Serialization.Metadata;
 
 namespace System.Text.Json.Serialization.Converters
 {
     /// <summary>
     /// Default base class implementation of <cref>JsonObjectConverter{T}</cref>.
     /// </summary>
-    internal class ObjectDefaultConverter<T> : JsonObjectConverter<T> where T : notnull
+    internal class ObjectDefaultConverter<T> : JsonObjectConverter<T>
```

Comment:
why the change to `notnull`?

Reply:
I'm not sure why this was originally added; affects new usage of generics added elsewhere. `T` can be null...